spec/models/api_azure_composite_web_search_spec.rb

```diff
@@ -17,26 +17,38 @@
     context 'when initialized with a Bing V2 key' do
       let(:api_key) { AzureEngine::DEFAULT_AZURE_HOSTED_PASSWORD }
 
+      before do
+        allow(AzureCompositeEngine).to receive(:new)
+      end
+
       it 'instantiates an AzureCompositeEngine' do
-        expect(AzureCompositeEngine).to receive(:new)
-        subject
+        search
+        expect(AzureCompositeEngine).to have_received(:new)
       end
 
       it 'uses module tag AZCW' do
-        expect(subject.default_module_tag).to eq('AZCW')
+        expect(search.default_module_tag).to eq('AZCW')
       end
     end
 
     context 'when initialized with a Bing V5 key' do
       let(:api_key) { BingV5HostedSubscriptionKey::BING_V5_SUBSCRIPTION_KEY }
 
+      before do
+        allow(BingV5WebEngine).to receive(:new)
+      end
+
       it 'instantiates a BingV5WebEngine' do
-        expect(BingV5WebEngine).to receive(:new)
-        subject
+        search
+        expect(BingV5WebEngine).to have_received(:new)
       end
 
       it 'uses module tag BV5W' do
-        expect(subject.default_module_tag).to eq('BV5W')
+        expect(search.default_module_tag).to eq('BV5W')
+      end
+
+      it 'returns a web hash' do
```

Comment:
Addition of this test (for code coverage) is the only real change here... the rest are CodeClimate inspired style updates.